bp_common/test/src/cpp-tests/src/constructor.cpp

```diff
@@ -0,0 +1,27 @@
+#include <iostream>
+using namespace std;
```

Comment:
Comment on what this test is checking?  Is there a way to verify it in code or is it just manual inspection?

Reply:
Applies for all tests

Reply:
Good point. I'll add some additional checks for some of the tests.